### PR TITLE
feat: update antd to v5.2.3

### PR DESCRIPTION
**Pull request by kv-chiu**

### CODE REVIEW

File: web/src/App.js [hunk 1]
```diff
@@ -44,6 +45,7 @@ import ChatEditPage from "./ChatEditPage";
 import ChatListPage from "./ChatListPage";
 import MessageListPage from "./MessageListPage";
 import MessageEditPage from "./MessageEditPage";
+import {Content} from "antd/es/layout/layout";
```
Comment (hsluoyz) [inline web/src/App.js]: Not this, see official example

File: web/src/App.js [hunk 2]
```diff
@@ -182,18 +185,22 @@ class App extends Component {
...
+            <LogoutOutlined className="rightdropdown-icon" />
+            {i18next.t("account:Sign Out")}
+          </div>
         </Menu.Item>
       </Menu>
     );
 
     return (
-      <Dropdown key="/rightDropDown" overlay={menu} className="rightDropDown">
+      <Dropdown key="/rightDropDown" overlay={menu} className="rightdropdown">
```
Comment (hsluoyz) [inline web/src/App.js]: Why change to `rightdropdown`?

File: web/src/App.less [hunk 3]
```diff
@@ -1,14 +1,12 @@
-@import '~antd/dist/antd.less';
-
-.App {
+.app {
   text-align: center;
 }
 
-.App-logo {
+.app-logo {
```
Comment (hsluoyz) [inline web/src/App.less]: Why change these?

File: web/src/Conf.js [hunk 4]
```diff
@@ -30,3 +30,10 @@ export const ForceLanguage = "";
 export const DefaultLanguage = "en";
 
 export const AppUrl = "";
+
+export const ThemeDefault = {
+  themeType: "default",
+  colorPrimary: "#5734d3",
```
Comment (hsluoyz) [inline web/src/Conf.js:36]: Does this work?

File: web/src/FileTree.js [hunk 5]
```diff
@@ -13,7 +13,24 @@
 // limitations under the License.
 
 import React from "react";
-import {Button, Col, DatePicker, Descriptions, Empty, Input, Modal, Popconfirm, Radio, Row, Select, Spin, Tooltip, Tree, Upload} from "antd";
+import {
+  Button,
```
Comment (hsluoyz) [inline web/src/FileTree.js]: Don't do this

**Review by hsluoyz — CHANGES_REQUESTED** [review]

**Comment by kv-chiu** [comment]
https://github.com/casbin/casibase/assets/132759132/096ab338-759f-41e3-bd72-1951d08d10c3

**Comment by kv-chiu** [comment]
https://github.com/casbin/casibase/assets/132759132/4b8f67b6-f876-4397-8890-8a5540f44172

**Review by hsluoyz — APPROVED** [review]

**Pull request merged**